data/content/environment_variables.yaml

```diff
@@ -98,11 +98,13 @@ variables:
 - name: BUILDKITE_BUILD_CREATOR
   desc: |
     The name of the user who created the build. May be **[unverified](#unverified-commits)**.
+
+    This will be set to the same value as the AUTHOR, unless the AUTHOR has not verified their email address with Buildkite, then it will fall back to an identity which it can be attributed to, often a bot user or the person whom merged a PR.
```

Comment:
A few questions first:

- Does "AUTHOR" here mean `BUILDKITE_BUILD_AUTHOR`?
- Is there anything helpful we can say about how we decide what fallback identity to attribute it to?
- How does the PR the person merged connect to this build? I'm wondering if it should be "the PR."